README.md

```diff
@@ -68,7 +68,7 @@ Follow these steps to quickly install the binaries for `foundry-zksync`:
    Begin by cloning the `foundry-zksync` repository from GitHub. This will download the latest version of the source code to your local machine.
 
    ```bash
-   git clone git@github.com:matter-labs/foundry-zksync.git
+   https://github.com/matter-labs/foundry-zksync.git
```

Comment:
Seemingly pointed in https://github.com/matter-labs/foundry-zksync/issues/363#issuecomment-2114906440 there's no precedence to change this. The other changes look good though.

Reply:
> Seemingly pointed in [#363 (comment)](https://github.com/matter-labs/foundry-zksync/issues/363#issuecomment-2114906440) there's no precedence to change this. The other changes look good though.

We are talking about public access. In this case, the HTTPS link looks preferable